docs/machine-learning/tutorials/tensorflow-transfer-learning.md

```diff
@@ -0,0 +1,337 @@
+
+[!code-csharp[MapValueToKey1](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#MapValueToKey1)]
+
+Your image processing pipeline uses pre-trained DNN featurizers for feature extraction. The images are loaded in memory as a Bitmap type. Next, you resize the images as the pre-trained model has a defined input image width and height. Finally, The `ImagePixelExtractingEstimator` extracts the pixels from the input images and converts them into a numeric vector. Add these image transforms as the next lines of code:
+
+[!code-csharp[ImageTransforms](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#ImageTransforms)]
+
+The `TensorFlowTransform` extracts specified outputs and scores a dataset using a pre-trained Tensorflow model. 
+
+[!code-csharp[ScoreTensorFlowModel](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#ScoreTensorFlowModel)]
+
+### Choose a learning algorithm
+
+To add the learning algorithm, call the `mlContext.MulticlassClassification.Trainers.LogisticRegression` wrapper method.  The `LogisticRegression` is appended to the `pipeline` and accepts the featurized `Title` and `Description` (`Features`) and the `Label` input parameters to learn from the historic data. You also need to map the label to the value to return to its original readable state. Do both of those actions with the following code:
+
+[!code-csharp[AddTrainer](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#AddTrainer)]
+
+You also need to map the predictedlabel to the predictedlabelvalue:
+
+[!code-csharp[MapValueToKey2](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#MapValueToKey2)]
+
+You train the model, <xref:Microsoft.ML.Data.TransformerChain%601>, based on the dataset that has been loaded and transformed. Once the estimator has been defined, you train your model using the <xref:Microsoft.ML.Data.EstimatorChain%601.Fit%2A> method while providing the already loaded training data. This returns a model to use for predictions. `pipeline.Fit()` trains the pipeline and returns a `Transformer` based on the `DataView` passed in. This DataView includes the `softmax2_pre_activation` property (also known as image features) and is produced by the ApplyTensorFlowGraph function. The experiment is not executed until the `.Fit()` method runs. Add this with the following code:
```

Comment:
What is softmax2_pre_activation? Explain this in the context of a scoring function for multi-class classification.